extensions/neo4j/deployment/src/main/java/io/quarkus/neo4j/deployment/NettyProcessor.java

```diff
@@ -0,0 +1,101 @@
+                "org.neo4j.driver.internal.shaded.io.netty.channel.socket.nio.NioSocketChannel"));
+        reflectiveClass
+                .produce(new ReflectiveClassBuildItem(false, false,
+                        "org.neo4j.driver.internal.shaded.io.netty.channel.socket.nio.NioServerSocketChannel"));
+        reflectiveClass.produce(new ReflectiveClassBuildItem(false, false, "java.util.LinkedHashMap"));
+
+        SubstrateConfigBuildItem.Builder builder = SubstrateConfigBuildItem.builder()
+                .addNativeImageSystemProperty("io.netty.noUnsafe", "true")
+                .addNativeImageSystemProperty("io.netty.leakDetection.level", "DISABLED")
+                .addRuntimeInitializedClass(
+                        "org.neo4j.driver.internal.shaded.io.netty.handler.ssl.JdkNpnApplicationProtocolNegotiator")
+                .addRuntimeInitializedClass(
+                        "org.neo4j.driver.internal.shaded.io.netty.handler.ssl.ReferenceCountedOpenSslEngine")
+                .addRuntimeInitializedClass(
+                        "org.neo4j.driver.internal.shaded.io.netty.handler.ssl.util.ThreadLocalInsecureRandom");
+        try {
+            Class.forName("org.neo4j.driver.internal.shaded.io.netty.handler.codec.http.HttpObjectEncoder");
+            builder.addRuntimeReinitializedClass("org.neo4j.driver.internal.shaded.io.netty.handler.codec.http2.Http2CodecUtil")
+                    .addRuntimeInitializedClass(
+                            "org.neo4j.driver.internal.shaded.io.netty.handler.codec.http.HttpObjectEncoder")
+                    .addRuntimeInitializedClass(
+                            "org.neo4j.driver.internal.shaded.io.netty.handler.codec.http2.DefaultHttp2FrameWriter")
+                    .addRuntimeInitializedClass(
+                            "org.neo4j.driver.internal.shaded.io.netty.handler.codec.http.websocketx.WebSocket00FrameEncoder");
+        } catch (ClassNotFoundException e) {
+            //ignore
+            log.debug("Not registering Netty HTTP classes as they were not found");
+        }
+        return builder //TODO: make configurable
+                .build();
+    }
+
+    @BuildStep
+    @Record(ExecutionTime.STATIC_INIT)
+    void createExecutors(BuildProducer<RuntimeBeanBuildItem> runtimeBeanBuildItemBuildProducer,
+            NettyTemplate template) {
+        //TODO: configuration
+        Supplier<Object> boss = template.createEventLoop(1);
+        Supplier<Object> worker = template.createEventLoop(0);
+
+        runtimeBeanBuildItemBuildProducer.produce(RuntimeBeanBuildItem.builder(EventLoopGroup.class)
+                .setSupplier(boss)
+                .setScope(ApplicationScoped.class)
+                .addQualifier(BossGroup.class)
+                .build());
+        runtimeBeanBuildItemBuildProducer.produce(RuntimeBeanBuildItem.builder(EventLoopGroup.class)
+                .setSupplier(worker)
+                .setScope(ApplicationScoped.class)
+                .build());
```

Comment:
Are these used?

Reply:
Yes. They have been copied from the Netty extension and do apply to the driver as well.